src/components/TopBarMenu/User/User.jsx

```diff
@@ -0,0 +1,7 @@
+import Images from "../../Images/Images";
+
+import "./User.scss";
+
+export default function User() {
+  return <Images imagetype="user" source="image-avatar.png" />;

```

Comment:
```suggestion
  return <Images imagetype="user" source="image-avatar.png" alt="users-avatar" />;
```
Once you had add alt on Images, you can pass via props. The value should be the meaning of the image.

If we're talking about an image product that came from the BE (Back-end) we can add its name:
example: 

```jsx
<Image source={src} alt={productName} />
```